### DISCUSSION TITLE
Reply by email not yet available? Possibility to start discussion by email.

### DISCUSSION THREAD

**Discussion by dev-zero** [community/community#4458] [Discussions]
While the notifications from the discussions say "Reply to this email directly." it doesn't seem to work (yet), is that correct?
I did reply to such a notification (with Thunderbird and the account it was sent to), but the message did not appear in the thread (or in the discussions in general).

Also, is there some way to start a new discussion (or issue) by mail? We have a couple of collaborators who really prefer to use email rather than having to login, and being able to interact with mail would make discussions much more inclusive for them.

**Comment by bczhc** [comment]
For the first one, may be related to https://github.com/orgs/community/discussions/101939? I also filed one.

**Comment by Unmesh100** [comment]
It sounds like you've encountered a bit of a snag with the email reply functionality. You're right; ideally, replying directly to an email notification should update the corresponding thread or discussion. If it isn't working yet, it might be a feature that's still in development or not fully implemented. Your best bet would be to check with the support or development team of the platform you're using to see if they have any updates or workarounds for this issue.

As for starting a new discussion or issue via email, this would indeed be a fantastic feature to make collaboration more seamless and inclusive, especially for those who prefer email over logging into a web interface. Unfortunately, most platforms don't support this out-of-the-box, but it's definitely worth bringing up as feedback to the development team. They might consider adding it if there's enough demand.

Have you looked into any third-party tools or integrations that might bridge this gap? Some email management or project management tools might offer such capabilities and could be a useful stopgap while the feature is being developed natively.
